tests/test_gitlab.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+import pytest
+from functools import wraps
+from utils import (
+    databricks_cli,
+    generated_project_dir,
+    parametrize_by_cloud,
+)
+
+@pytest.mark.parametrize(
+    "cicd_platform", ["gitlab"]
+)
+@pytest.mark.parametrize(
+    "setup_cicd_and_project,include_feature_store,include_mlflow_recipes,include_models_in_unity_catalog",
+    [
+        ("CICD_and_Project", "no", "no", "yes"),
```

Comment:
Do we only allow GitLab with Models in UC?

Reply:
I've only tested it with models in UC. But there is nothing stopping us from allowing all cases. shall I copy all the option we use for github?

Reply:
added all cases